src/hnasko_lab_to_nwb/embargo_2025/convert_session.py

```diff
@@ -119,7 +122,8 @@ def session_to_nwb(
     path_expander = LocalPathExpander()
     # Expand paths and extract metadata
     metadata_list = path_expander.expand_paths(source_data_spec)
```

Comment:
Neat! I didn't know we had this, but this will be useful for my future conversions.